plugins/kubernetes-actions/src/actions/createKubernetesNamespace.ts

```diff
@@ -68,7 +68,7 @@ const getUrlFromClusterRef = async (
 const validateUrl = (url: string | undefined) => {
   try {
     // eslint-disable-next-line no-new
-    new URL(url!);
+    new URL(url as string);
```

Comment:
Can you please create a test case for this? https://github.com/janus-idp/backstage-plugins/blob/main/plugins/kubernetes-actions/src/actions/createKubernetesNamespace.test.ts

I'm not sure casting this to string is the best option. What about:
```suggestion
    new URL(url || "");
```

or even better, in the signature:

```diff
- const validateUrl = (url: string | undefined) => {
+ const validateUrl = (url: string | undefined = "") => {
```

Reply:
Thanks for the suggeation @tumido. I believe `const validateUrl = (url: string | undefined = "") => {` would be best for this use case as it is by default assigning the empty string to undefined and we don't need to check/set the type of `url`.

Reply:
I will also write the tests for empty string `""` and the `undefended` value to clarify the above.